std_msgs/Header header
int32 xian_dj_stewart_platform_input_alpha_positive_cmd
int32 xian_dj_stewart_platform_input_alpha_negative_cmd
int32 xian_dj_stewart_platform_input_beta_positive_cmd
int32 xian_dj_stewart_platform_input_beta_negative_cmd
int32 xian_dj_stewart_platform_input_gamma_positive_cmd
int32 xian_dj_stewart_platform_input_gamma_negative_cmd
int32 xian_dj_stewart_platform_input_x_positive_cmd
int32 xian_dj_stewart_platform_input_x_negative_cmd
int32 xian_dj_stewart_platform_input_y_positive_cmd
int32 xian_dj_stewart_platform_input_y_negative_cmd
int32 xian_dj_stewart_platform_input_z_positive_cmd
int32 xian_dj_stewart_platform_input_z_negative_cmd